pegasus/helpers/hourofcode_helpers.rb

```diff
@@ -149,9 +149,13 @@ def campaign_date(format)
   end
 end
 
+def all_unique_events
+  DB[:forms].where(kind: 'HocSignup2015').select(:name, :email).distinct
+end
+
 def company_count
   company_count = 0;
-  DB[:forms].where(kind: 'HocSignup2015').each do |i|
+  all_unique_events.each do |i|
     data = JSON.parse(i[:data])
     if data['hoc_company_s'] == @company
```

Comment:
does this still work? I don't think the all_unique_events query that you're using here selects the data field, so I don't think you'll have an i[:data]